vector/src/main/java/im/vector/app/features/settings/devices/v2/list/CurrentSessionView.kt

```diff
@@ -39,21 +39,14 @@ class CurrentSessionView @JvmOverloads constructor(
         views = ViewCurrentSessionBinding.bind(this)
     }
 
-    fun update(accountCrossSigningIsTrusted: Boolean, legacyMode: Boolean) {
-        renderDeviceType()
-        renderVerificationStatus(accountCrossSigningIsTrusted, legacyMode)
+    fun update(currentDeviceInfo: DeviceFullInfo) {
+        renderDeviceInfo(currentDeviceInfo.deviceInfo.displayName ?: "")
```

Comment:
We could use the `currentDeviceInfo.deviceInfo.displayName.orEmpty()` syntax to make things clearer.

Reply:
Nice idea, done.